code/modules/mob/living/simple_animal/borer/borer_html.dm

```diff
@@ -0,0 +1,129 @@
+
+/mob/living/simple_animal/borer/proc/get_html_template(content)
+	var/html = {"<!DOCTYPE HTML PUBLIC \"-//W3C//DTD HTML 4.01 Transitional//EN\" \"http://www.w3.org/TR/html4/loose.dtd\">
+		<html>
```

Comment:
@PKPenguin321 the amount of motivation they have to finish the PR is none of my problem

if you want to port code you have to put in the effort to make it meet our standards.